SumoForkClass.php

```diff
@@ -117,7 +117,7 @@ public function initErrbit()
                     $this->debug = $this->getContainer()->hasParameter('fork.debug');
                 }
                 $this->errbitApiKey = $this->getContainer()->getParameter('sumo.errbit_api_key');
-            } catch (\Symfony\Component\DependencyInjection\Exception\ParameterNotFoundException $e) {
```

Comment:
Ok, just asked to make sure that we wouldn't get other exceptions by changing it